contracts/modules/cover/Cover.sol

```diff
@@ -779,6 +779,19 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     return (1 << coverAsset) & assetsBitMap > 0;
   }
 
+  function getCapacityRatios(uint[] calldata productIds) public view returns (uint _globalCapacityRatio, uint[] memory initialPrices, uint[] memory capacityReductionRatios) {
```

Comment:
1. Split the definition on multiple lines:
```solidity
  function getCapacityRatios(uint[] calldata productIds) public view returns (
    uint _globalCapacityRatio,
    uint[] memory initialPrices,
    uint[] memory capacityReductionRatios
  ) {
```
2. This function returns both ratios and the initial price, the name does not suggest that

Reply:
3. We need this function to return the `GLOBAL_MIN_PRICE_RATIO` as well as we'll need it when setting target price in StakingPool